Framework/Geometry/src/Crystal/OrientedLattice.cpp

```diff
@@ -321,10 +321,7 @@ void OrientedLattice::recalculate() {
       (da[5] > da[4] + da[3])) {
     throw std::invalid_argument("Invalid angles");
   }
-  UnitCell::calculateG();
```

Comment:
The `if` statement above is also in `UnitCell::recalculate()` Can we also remove it?